TaskEditor/EventViewerControl.cs

```diff
@@ -39,9 +40,9 @@ public TaskEvent TaskEvent
 					computerText.Text = taskEvent.EventRecord.MachineName;
 					detailsText.Text = taskEvent.EventRecord.FormatDescription();
 					eventIdText.Text = taskEvent.EventRecord.Id.ToString();
-					System.Collections.Generic.List<string> keywords = new System.Collections.Generic.List<string>();
-					foreach (string s in taskEvent.EventRecord.KeywordsDisplayNames)
-						keywords.Add(s);
+					System.Collections.Generic.List<string> keywords =

```

Comment:
Will you change lines 43-46 to the following?
```suggestion
keywordsText.Text = string.Join(", ", taskEvent.EventRecord.KeywordsDisplayNames.ToArray());
```

Reply:
I just added initial capacity for the List.

Reply:
I saw that. My suggestion was based on not even needing to create the list in the first place and directly supplying `string.Join` with a string array created via `Linq`.